packages/block-editor/src/components/page-template-picker/templates/portfolio.native.js

```diff
@@ -0,0 +1,234 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+const Portfolio = {
+	// translators: title for "Portfolio" page template
+	name: __( 'Portfolio' ),
+	icon: '🎨',
+	content: [
+		{
+			name: 'core/paragraph',
+			attributes: {
+				align: 'left',
+				// translators: sample content for "Portfolio" page template
+				content: __(
+					'My portfolio showcases various projects created throughout my career. See my contact information below and get in touch.'
+				),
+			},
+		},
+		{
+			name: 'core/spacer',
+			attributes: {
+				height: 24,
+			},
+		},
+		{
+			name: 'core/heading',
+			attributes: {
+				align: 'left',
+				// translators: sample content for "Portfolio" page template
+				content: __( 'Project Name' ),
+				level: 2,
+			},
+		},
+		{
+			name: 'core/gallery',
+			attributes: {
+				images: [
+					{
+						url:
+							'https://a8ctm1.files.wordpress.com/2019/08/scatter-1.jpg?w=640',
+						link:
+							'https://a8ctm1.wordpress.com/portfolio/scatter-3/',
+						id: '658',
```

Comment:
This could be problematic. I imagine those IDs are for the site where the templates were created, but it could break things on any other site.

Also perhaps we should avoid the links as week, it's one thing that we are hot linking images from a demo site, but having them link to that site as well would be unexpected and confusing.

Reply:
Interestingly enough, if I remove the links and ids from the template, when you try to add an image to that gallery, it replaces all the existing images. I'm not entirely sure if that's good or bad 😁

Reply:
😅You're right, I've just tested that myself. Were there any discussions about these cases? Blocks that have media content: `Gallery`, `Media & Text` etc.?

Reply:
Not that I'm aware of, I think we didn't consider it could be an issue. I'd say removing the IDs and links is likely the least harmful option. It might be a bit confusing if they disappear when the user adds a new image, but they are meant to be placeholders, so maybe not that confusing? cc @pinarol @iamthomasbishop

Reply:
Why don't we loose the IDs but keep the links? When we insert an image from a web url we don't have any ID at all, so it should be a valid case?

```
<!-- wp:image {"sizeSlug":"large"} -->
<figure class="wp-block-image size-large"><img src="https://a8ctm1.files.wordpress.com/2019/08/scatter-1.jpg" alt=""/></figure>
<!-- /wp:image -->
```

Reply:
> Interestingly enough, if I remove the links and ids from the template, when you try to add an image to that gallery, it replaces all the existing images. 

This looks like a bug? It should be happening independent from the SPTs

Reply:
> Why don't we loose the IDs but keep the links? When we insert an image from a web url we don't have any ID at all, so it should be a valid case?

This works! Forgot about linking images directly by URL, so there won't be any issues if we remove the IDs

>> Interestingly enough, if I remove the links and ids from the template, when you try to add an image to that gallery, it replaces all the existing images.

>This looks like a bug? It should be happening independent from the SPTs

Hmm I don't know if its a bug or just an edge case. I tried to do that on the Web editor and the gallery removes the images when you add a new one as well. (Using this template without IDs)

Reply:
Ok thanks for trying on web.

Reply:
So do we have a consensus about moving fwd with only links? @koke @geriux Anything left to prevent merging this PR?

Reply:
All good from my side =)

Reply:
Looking good to me too